std_msgs/Int8[] motor_amp
std_msgs/Int8[] battery_voltage
